packages/dev/test/ServerTestEnvironment.ts

```diff
@@ -277,8 +277,11 @@ export class ServerTestEnvironment {
       const mergedTransactionDetail = Object.assign({}, gsnTransactionDetails, overrideDetails)
       // do not 'return await' here as it will defer executing the 'finally' block and enable re-stubbing
       // (will crash on 'let x = [createRelayHttpRequest(), createRelayHttpRequest()]')
-      // eslint-disable-next-line @typescript-eslint/return-await
-      return this.relayClient._prepareRelayHttpRequest(relayInfo, mergedTransactionDetail)
+      // eslint-disable-next-line @typescript-eslint/return-await,@typescript-eslint/promise-function-async
```

Comment:
is this delibarate? you return (and actually, call "finally" too) before the async function is executed.
why not do "return  __await__ this.relayClient._prepareRelayRequest" ?